### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by tkurki**
See individual commits

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -59,21 +32,21 @@ const CONFIG_SCHEMA = {
       properties: {
         routes: {
           type: 'boolean',
-          title: 'ROUTES'
+          title: 'ROUTES',
```
Comment (panaaj) [inline src/index.ts:35]: Trailing commas seem to have been added... is this a `prettier` formatting issue?

File: src/lib/utils.ts [hunk 2]
```diff
@@ -1,187 +1,173 @@
...
+      break
+  }
+  return ok
+}
+
+/** Apply filters to Resource entry
+ * returns: true if entry should be included in results **/
+export function passFilter(res: any, type: string, params: any) {
+  let ok = true
+  if (params.region) {
```
Comment (panaaj) [inline src/lib/utils.ts:65]: should be `params.href` now for v2 of the spec.

File: src/lib/utils.ts [hunk 3]
```diff
@@ -1,187 +1,173 @@
...
+    // ** check is attached to region
+    // console.error(`check region: ${params.region}`);
+    if (typeof res.region === 'undefined') {
```
Comment (panaaj) [inline src/lib/utils.ts:68]: update to `res.href` for `v2`

File: src/lib/utils.ts [hunk 4]
```diff
@@ -1,187 +1,173 @@
...
+      ok = ok && false
+    } else {
+      // deconstruct resource region value
+      const ha = res.region.split('/')
```
Comment (panaaj) [inline src/lib/utils.ts:72]: update to `res.href` for `v2`

File: src/lib/utils.ts [hunk 5]
```diff
@@ -1,187 +1,173 @@
...
+      const hType: string =
+        ha.length === 1
+          ? 'regions'
+          : ha.length > 2
+          ? ha[ha.length - 2]
+          : 'regions'
+      const hId = ha.length === 1 ? ha[0] : ha[ha.length - 1]
+
+      // deconstruct param.region value
```
Comment (panaaj) [inline src/lib/utils.ts:81]: update to `res.href` for `v2`

File: src/lib/utils.ts [hunk 6]
```diff
@@ -1,187 +1,173 @@
...
+ * returns: true if entry should be included in results **/
+export function passFilter(res: any, type: string, params: any) {
+  let ok = true
+  if (params.region) {
+    // ** check is attached to region
+    // console.error(`check region: ${params.region}`);
+    if (typeof res.region === 'undefined') {
+      ok = ok && false
+    } else {
+      // deconstruct resource region value
```
Comment (panaaj) [inline src/lib/utils.ts:71]: update to `res.href` for `v2`

File: src/lib/filestorage.ts [hunk 7]
```diff
@@ -1,212 +1,212 @@
...
-}
+import { constants } from 'fs'
+import {
+  access,
+  mkdir,
+  readdir,
+  readFile,
+  stat,
+  unlink,
+  writeFile,
```
Comment (panaaj) [inline src/lib/filestorage.ts:9]: trailing comma

File: src/lib/filestorage.ts [hunk 8]
```diff
@@ -1,212 +1,212 @@
...
+
+    if (r.value === null) {
+      // ** delete file **
+      try {
+        await unlink(p)
+        this.debug(`** DELETED: ${r.type} entry ${fname} **`)
+        return
+      } catch (error) {
+        console.error('Error deleting resource!')
+        ;(error as Error).message = 'Error deleting resource!'
```
Comment (panaaj) [inline src/lib/filestorage.ts:170]: leading semicolon

**Review by panaaj — APPROVED** [review]
Looks OK except for the change from `region` to `href` in the parameter passing due to the update in Notes schema..
Just seems to be some formatting issues with some trailing commas.

File: src/index.ts (hunk 1 above)
Comment (tkurki) [reply src/index.ts]: yep, fixed

**Comment by tkurki** [comment]
One more try, hope everything's good now.

**Pull request merged**